src/main/java/org/spongepowered/api/text/Text.java

```diff
@@ -933,6 +936,37 @@ public static Text of(Object... objects) {
         return builder.build();
     }
 
+    public static Optional<Text> get(Dictionary dictionary, String key, Locale locale) {
```

Comment:
You'll always need a simple string from time to time..... Sending messages is only one area where Text is used, it's used in a lot of other places as well.